### PR TITLE
UHF-5923: Job listing search fallback

### PR DESCRIPTION
**Pull request by teroelonen**
# [UHF-5923](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5923)

## What was done

* Added job listing search fallback view
* Created job listing search paragraph and added it to landing page

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-5923_job_listing_search_fallback`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [x] Create a landing page and add the job listing search paragraph to it. Save the node.
* [x] Go to view the node and make sure the fallback listing works without javascript and looks good. Notice that the design of the job listing card has changed and it affects the "You might be interested block" on single job listing page as well.
* [x] Check that code follows our standards

## Designers review

* [ ] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Sami Haikonen)

### CODE REVIEW

File: public/modules/custom/helfi_rekry_content/helfi_rekry_content.module [hunk 1]
```diff
@@ -243,3 +244,44 @@ function _helfi_rekry_content_lookup_organization_tid($external_id) {
...
+  if ((bool) $entity->get('field_has_hero')->value) {
+    if (
+      in_array($entity->getType(), ['landing_page', 'page']) &&
+      isset($entity->get('field_content')->first()->entity)
+    ) {
+
+      $first_paragraph_gray = &$variables['content']['hero_block']['#first_paragraph_grey'];
+
+      // Check if the content field first paragraph is District and project
+      // search or District listing and add classes accordingly.
```
Comment (Jussiles) [inline public/modules/custom/helfi_rekry_content/helfi_rekry_content.module:280]: Remove the District and project related comments :)

File: public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_job-listing-search.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.component--job-listing-search {
+  background-color: $color-silver-light;
+}
+
+// If the district and project search is just before footer it needs to function the same way as unit search.
```
Comment (Jussiles) [inline public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_job-listing-search.scss:5]: Remove the District and project related comments also from here :)

**Review by Jussiles — APPROVED** [review]
Looks and work good! 🦜 Fix the comments that I added and you can merge.

File: public/modules/custom/helfi_rekry_content/helfi_rekry_content.module (hunk 1 above)
Comment (teroelonen) [reply public/modules/custom/helfi_rekry_content/helfi_rekry_content.module:280]: Nice catch! Fixed :)

File: public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_job-listing-search.scss (hunk 2 above)
Comment (teroelonen) [reply public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_job-listing-search.scss:5]: Nice catch! Fixed :)

**Pull request merged**